### PR TITLE
add derived field for power of a field

### PR DESCRIPTION
**Pull request by nicolemarsaglia**
derived_field = field^exponent

### CODE REVIEW

**Comment by nicolemarsaglia** [comment]
@cyrush I'm not really understanding the cuda error I'm getting on the failing pipeline and how my changes would have caused it. Any guesses or have an idea where I should start digging? 

```
CMake Warning (dev) in CMakeLists.txt:
  Policy CMP0104 is not set: CMAKE_CUDA_ARCHITECTURES now detected for NVCC,
  empty CUDA_ARCHITECTURES not allowed.  Run "cmake --help-policy CMP0104"
  for policy details.  Use the cmake_policy command to set the policy and
  suppress this warning.

  CUDA_ARCHITECTURES is empty for target "ascent_render_example".
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Generating done (0.0s)
-- Build files have been written to: /__w/1/s/install/examples/ascent/using-with-cmake/_test_build
/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/cmake-3.26.3-ijtycunvtuswebyghckzo577zakhrhx4/bin/cmake -S/__w/1/s/install/examples/ascent/using-with-cmake -B/__w/1/s/install/examples/ascent/using-with-cmake/_test_build --check-build-system CMakeFiles/Makefile.cmake 0
/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/cmake-3.26.3-ijtycunvtuswebyghckzo577zakhrhx4/bin/cmake -E cmake_progress_start /__w/1/s/install/examples/ascent/using-with-cmake/_test_build/CMakeFiles /__w/1/s/install/examples/ascent/using-with-cmake/_test_build//CMakeFiles/progress.marks
make  -f CMakeFiles/Makefile2 all
make[1]: Entering directory '/__w/1/s/install/examples/ascent/using-with-cmake/_test_build'
make  -f CMakeFiles/ascent_render_example.dir/build.make CMakeFiles/ascent_render_example.dir/depend
make[2]: Entering directory '/__w/1/s/install/examples/ascent/using-with-cmake/_test_build'
cd /__w/1/s/install/examples/ascent/using-with-cmake/_test_build && /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/cmake-3.26.3-ijtycunvtuswebyghckzo577zakhrhx4/bin/cmake -E cmake_depends "Unix Makefiles" /__w/1/s/install/examples/ascent/using-with-cmake /__w/1/s/install/examples/ascent/using-with-cmake /__w/1/s/install/examples/ascent/using-with-cmake/_test_build /__w/1/s/install/examples/ascent/using-with-cmake/_test_build /__w/1/s/install/examples/ascent/using-with-cmake/_test_build/CMakeFiles/ascent_render_example.dir/DependInfo.cmake --color=
make[2]: Leaving directory '/__w/1/s/install/examples/ascent/using-with-cmake/_test_build'
make  -f CMakeFiles/ascent_render_example.dir/build.make CMakeFiles/ascent_render_example.dir/build
make[2]: Entering directory '/__w/1/s/install/examples/ascent/using-with-cmake/_test_build'
[ 33%] Building CXX object CMakeFiles/ascent_render_example.dir/ascent_render_example.cpp.o
/usr/bin/c++ -DCAMP_HAVE_CUDA -DH5_BUILT_AS_DYNAMIC_LIB -DVTKMDIY_MPI_AS_LIB -DVTKMDIY_MPI_STATIC_BUILD -DVTKMDIY_NO_THREADS -D_FILE_OFFSET_BITS=64 -D_GNU_SOURCE -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -D_POSIX_C_SOURCE=200809L -isystem /__w/1/s/install/include -isystem /__w/1/s/install/include/ascent -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/conduit-0.8.8-xechkwu4wvedgmvplho5hdnu6aailws6/include -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/conduit-0.8.8-xechkwu4wvedgmvplho5hdnu6aailws6/include/conduit -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/occa-1.2.0-22kje22k4wfvykwjow3ogjo4lmxdab2v/include -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/mfem-4.5.2-t4cqchvjnwfqc6hjxinyzewxhvd3inv6/include -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/caliper-2.9.0-djv2lcp2znknckyf2a7hhkjuya5vgjnz/include -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/hdf5-1.8.22-ekiupszjcpngbn7whbp66orx543k6gnk/include -isystem /usr/local/cuda/include -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/raja-2022.10.4-kw7r56wnkbw6ru5d2zec2kl67jxvgbhc/include -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/cub-2.1.0-iiodai5fpmttkwuiueusfaxlo3ornt5b/include -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/camp-2022.10.1-65uyczen3o2a2tln6ftbxifdk6mwsn22/include -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/umpire-2022.03.1-ruqsp3s2wgo5cgez7moexkxs4omezcmc/include -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/include/vtkm-2.1/vtkm/thirdparty/optionparser -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/include/vtkm-2.1/vtkm/thirdparty/diy -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/include/vtkm-2.1/vtkm/thirdparty/diy/vtkmdiy/include -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/include/vtkm-2.1/vtkm/thirdparty/diy/vtkmdiy/include/vtkmdiy/mpi -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/include/vtkm-2.1/vtkm/thirdparty/lcl/vtkmlcl -isystem /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/include/vtkm-2.1 -isystem /__w/1/s/install/include/ascent/vtkh -isystem /usr/include/x86_64-linux-gnu/mpich -isystem /__w/1/s/install/include/ascent/dray -isystem /__w/1/s/install/include/ascent/apcomp -O3 -DNDEBUG -I/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/hypre-2.28.0-kwkt2ksi4qydjb7dxkpzwjs2iljyb5hh/include -I/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/metis-5.1.0-il2qmgyzvshfuadxlkeiv3vbm5hh6xqn/include -I/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/conduit-0.8.8-xechkwu4wvedgmvplho5hdnu6aailws6/include/conduit -I/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/conduit-0.8.8-xechkwu4wvedgmvplho5hdnu6aailws6/include -I/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/hdf5-1.8.22-ekiupszjcpngbn7whbp66orx543k6gnk/include -I/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/zlib-1.2.13-aimvv2vbiretkadj3mumnon3b7m6chrn/include -ffunction-sections -MD -MT CMakeFiles/ascent_render_example.dir/ascent_render_example.cpp.o -MF CMakeFiles/ascent_render_example.dir/ascent_render_example.cpp.o.d -o CMakeFiles/ascent_render_example.dir/ascent_render_example.cpp.o -c /__w/1/s/install/examples/ascent/using-with-cmake/ascent_render_example.cpp
[ 66%] Linking CUDA device code CMakeFiles/ascent_render_example.dir/cmake_device_link.o
/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/cmake-3.26.3-ijtycunvtuswebyghckzo577zakhrhx4/bin/cmake -E cmake_link_script CMakeFiles/ascent_render_example.dir/dlink.txt --verbose=1
/usr/local/cuda/bin/nvcc -forward-unknown-to-host-compiler -O3 -DNDEBUG -Xcompiler=-fPIC -Wno-deprecated-gpu-targets -shared -dlink --options-file CMakeFiles/ascent_render_example.dir/deviceObjects1.rsp -o CMakeFiles/ascent_render_example.dir/cmake_device_link.o --options-file CMakeFiles/ascent_render_example.dir/deviceLinkLibs.rsp
[100%] Linking CXX executable ascent_render_example
/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/cmake-3.26.3-ijtycunvtuswebyghckzo577zakhrhx4/bin/cmake -E cmake_link_script CMakeFiles/ascent_render_example.dir/link.txt --verbose=1
/usr/bin/c++ -O3 -DNDEBUG -L/usr/local/cuda//lib64 -L/usr/local/cuda/lib64 -Wl,-Bsymbolic-functions CMakeFiles/ascent_render_example.dir/ascent_render_example.cpp.o CMakeFiles/ascent_render_example.dir/cmake_device_link.o -o ascent_render_example   -L/usr/local/cuda/targets/x86_64-linux/lib  -Wl,-rpath,/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/occa-1.2.0-22kje22k4wfvykwjow3ogjo4lmxdab2v/lib:/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/zlib-1.2.13-aimvv2vbiretkadj3mumnon3b7m6chrn/lib:/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/conduit-0.8.8-xechkwu4wvedgmvplho5hdnu6aailws6/lib:/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/hdf5-1.8.22-ekiupszjcpngbn7whbp66orx543k6gnk/lib:/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/caliper-2.9.0-djv2lcp2znknckyf2a7hhkjuya5vgjnz/lib:/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/adiak-0.2.2-loxmtwsal5nybbblro7zbg3cjqxlml7w/lib /__w/1/s/install/lib/libascent.a /__w/1/s/install/lib/libascent_flow.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/occa-1.2.0-22kje22k4wfvykwjow3ogjo4lmxdab2v/lib/libocca.so /usr/local/cuda-12.1/targets/x86_64-linux/lib/stubs/libcuda.so /__w/1/s/install/lib/librover.a /__w/1/s/install/lib/libdray.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/raja-2022.10.4-kw7r56wnkbw6ru5d2zec2kl67jxvgbhc/lib/libRAJA.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/umpire-2022.03.1-ruqsp3s2wgo5cgez7moexkxs4omezcmc/lib/libumpire.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/mfem-4.5.2-t4cqchvjnwfqc6hjxinyzewxhvd3inv6/lib/libmfem.a -Wl,-rpath,/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/hypre-2.28.0-kwkt2ksi4qydjb7dxkpzwjs2iljyb5hh/lib -Wl,-rpath,/usr/lib/x86_64-linux-gnu/lapack -Wl,-rpath,/usr/lib/x86_64-linux-gnu/blas -L/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/hypre-2.28.0-kwkt2ksi4qydjb7dxkpzwjs2iljyb5hh/lib -L/usr/lib/x86_64-linux-gnu/lapack -L/usr/lib/x86_64-linux-gnu/blas -lHYPRE -llapack -lblas -Wl,-rpath,/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/metis-5.1.0-il2qmgyzvshfuadxlkeiv3vbm5hh6xqn/lib -L/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/metis-5.1.0-il2qmgyzvshfuadxlkeiv3vbm5hh6xqn/lib -lmetis -Wl,-rpath,/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/conduit-0.8.8-xechkwu4wvedgmvplho5hdnu6aailws6/lib -Wl,-rpath,/lib/x86_64-linux-gnu -Wl,-rpath,/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/hdf5-1.8.22-ekiupszjcpngbn7whbp66orx543k6gnk/lib -L/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/conduit-0.8.8-xechkwu4wvedgmvplho5hdnu6aailws6/lib -L/lib/x86_64-linux-gnu -L/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/hdf5-1.8.22-ekiupszjcpngbn7whbp66orx543k6gnk/lib -lconduit -lconduit_relay -lconduit_blueprint -ldl -lhdf5 -lrt -Wl,-rpath,/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/zlib-1.2.13-aimvv2vbiretkadj3mumnon3b7m6chrn/lib -L/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/zlib-1.2.13-aimvv2vbiretkadj3mumnon3b7m6chrn/lib -lz /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/zlib-1.2.13-aimvv2vbiretkadj3mumnon3b7m6chrn/lib/libz.so /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/camp-2022.10.1-65uyczen3o2a2tln6ftbxifdk6mwsn22/lib/libcamp.a /__w/1/s/install/lib/libapcomp.a /__w/1/s/install/lib/libvtkh_rendering.a /__w/1/s/install/lib/libvtkh_filters.a /__w/1/s/install/lib/libvtkh_compositing.a /__w/1/s/install/lib/libvtkh_core.a /__w/1/s/install/lib/libvtkh_utils.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/conduit-0.8.8-xechkwu4wvedgmvplho5hdnu6aailws6/lib/libconduit_relay.so /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/hdf5-1.8.22-ekiupszjcpngbn7whbp66orx543k6gnk/lib/libhdf5.so.10.4.0 /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/conduit-0.8.8-xechkwu4wvedgmvplho5hdnu6aailws6/lib/libconduit_blueprint.so /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/conduit-0.8.8-xechkwu4wvedgmvplho5hdnu6aailws6/lib/libconduit.so /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/caliper-2.9.0-djv2lcp2znknckyf2a7hhkjuya5vgjnz/lib/libcaliper.so.2.9.0 /usr/lib/x86_64-linux-gnu/libmpichcxx.so /usr/lib/x86_64-linux-gnu/libmpich.so /__w/1/s/install/lib/libascent_lodepng.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_connected_components-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_contour-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_vector_analysis-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_mesh_info-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_field_conversion-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_field_transform-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_flow-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_multi_block-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_resampling-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_density_estimate-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_geometry_refinement-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_scalar_topology-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_zfp-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_rendering-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_entity_extraction-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_clean_grid-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_image_processing-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_filter_core-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_worklet-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_io-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkm_cont-2.1.a /uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/vtk-m-2.1.0-scw3zitdvpqwstaqhdbpyacxe5mkylpd/lib/libvtkmdiympi_nompi.a -lrt -lpthread /usr/local/cuda/lib64/libcudart_static.a -ldl /usr/lib/x86_64-linux-gnu/librt.so -lcudadevrt -lcudart_static -lrt -lpthread -ldl -Wl,-rpath-link,/uberenv_libs/spack/opt/spack/linux-ubuntu20.04-zen2/gcc-9.4.0/adiak-0.2.2-loxmtwsal5nybbblro7zbg3cjqxlml7w/lib 
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/ascent_render_example.dir/build.make:215: ascent_render_example] Error 1
make[2]: Leaving directory '/__w/1/s/install/examples/ascent/using-with-cmake/_test_build'
make[1]: Leaving directory '/__w/1/s/install/examples/ascent/using-with-cmake/_test_build'
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/ascent_render_example.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

File: src/examples/using-with-cmake/CMakeLists.txt [hunk 1]
```diff
@@ -49,5 +49,6 @@ target_link_libraries(ascent_render_example ascent::ascent)
 # we need to make sure CUDA_RESOLVE_DEVICE_SYMBOLS is on for our target
 # (it propgates some way magically in 3.14, but not in 3.21)
 if(CMAKE_CUDA_COMPILER)
-    set_property(TARGET ascent_render_example PROPERTY CUDA_RESOLVE_DEVICE_SYMBOLS ON)
+	set_property(TARGET ascent_render_example PROPERTY CUDA_RESOLVE_DEVICE_SYMBOLS ON)
+	#set_property(TARGET ascent_render_example PROPERTY CUDA_ARCHITECTURES "70")
```
Comment (mclarsen) [inline src/examples/using-with-cmake/CMakeLists.txt]: I think this is the line causing the cuda error you were asking about.

Comment (nicolemarsaglia) [reply src/examples/using-with-cmake/CMakeLists.txt]: @mclarsen Yep! Something changed with Azure so we were running out of build space. Changing the build type from Release to MinSizeRel helped: https://github.com/Alpine-DAV/ascent/blob/ecc36b4e6d2987ae2b55f2334c3abca737f0163b/azure-pipelines.yml#L449

**Comment by nicolemarsaglia** [comment]
Uh oh @cyrush 

merged in develop to restart the ci checks and while they are green, now it says that no files have been changed... How did I do this?

**Comment by cyrush** [comment]
@nicolemarsaglia  that is very strange -- it still shows all of the commits but doesn't think anything has changed. 
Let me see what I can understand from the branch.

**Comment by nicolemarsaglia** [comment]
> @nicolemarsaglia that is very strange -- it still shows all of the commits but doesn't think anything has changed. Let me see what I can understand from the branch.

Thanks @cyrush ! No rush!

**Comment by cyrush** [comment]
Looks like in the past you may have merged from this branch into the cinema output path branch:
```
 Merge branch 'task/2024_02_add_power_df' into task/2024_02_add_cinema_output_path
```

That branch was merged to develop with PR #1256.

So things may truly already be on develop?
Can you spot check a few of the important/unique changes and see if they are there?

**Comment by nicolemarsaglia** [comment]
@cyrush Oh interesting! I vaguely recall that. I would not have expected these side effects. Thanks for figuring this out, looks like it's all there from that merge.

**Pull request closed without merging**